docker-compose.yaml

```
@@ -67,4 +67,6 @@ networks:
     name: jiran-tetangga-networks
 
 volumes:
-  db_data:
\ No newline at end of file
+
+  db_data:
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)